framework/src/application/node/node.ts

```diff
@@ -264,51 +248,35 @@ export class Node {
 		}
 	}
 
+	public get networkIdentifier(): Buffer {
+		return this._networkIdentifier;
+	}
+
 	// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types,@typescript-eslint/explicit-function-return-type
 	public get actions() {
 		return {
```

Comment:
what about the modules actions?

Reply:
Ill create another issue for this. i havent figured out how exactly it should be setup

Reply:
https://github.com/LiskHQ/lisk-sdk/issues/5639